default/scripting/specials/planet/ECCENTRIC_ORBIT.focs.txt

```diff
@@ -7,8 +7,9 @@ Special
     location = And [
         Planet
         Not Capital
-        Not Planet type = [Asteroids GasGiant]
+        Not Planet type = [Asteroids GasGiant]  // why not?
```

Comment:
probably because these are not standard environments and it was not possible to settle these. maybe thats not valid anymore. remove the comment and go to the forums.

Reply:
aye.